src/frontends/paddle/tests/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # PaddlePaddle - generate test models
 paddlepaddle==2.4.0
 gast==0.3.3
-numpy>=1.16.6,<=1.23.4
+numpy>=1.16.6,<1.24.2
```

Comment:
```suggestion
numpy>=1.16.6,<=1.24.0
```

Reply:
To be aligned with other reqs files. It looks that this particular file was missed when we transferred to 1.24 version.

Reply:
@ilya-lavrenov, can you please commit my changes here? Looks not all are aligned now.

Reply:
Looks like we have mistake here and proper upper bound is <1.25.0?